tests/commands/test_sre.py

```diff
@@ -111,6 +113,7 @@ def test_teardown(
         mock_sre_config_from_remote,  # noqa: ARG002
         mock_sre_project_manager_teardown_then_exit,  # noqa: ARG002
     ) -> None:
+        mocker.patch.object(Confirm, "ask", return_value="yes")
```

Comment:
We can use the runner to [test input](https://typer.tiangolo.com/tutorial/testing/#call-pytest).